cypress/integration/article_spec.js

```diff
@@ -81,4 +81,12 @@ describe('News Article', () => {
       styleTag.should('contain', font);
     });
   });
+
+  // Testing the actual fetch is not currently possible
+  it('should have script to fetch bundle', () => {
+    cy.get('script')
+      .last()
```

Comment:
The last script object might not always be the bundle, so this is brittle. We should assert that the `body` element contains a script with the specified attributes.